sdk/tables/azure-data-tables/samples/tables_getting_started.cpp

```diff
@@ -33,17 +33,17 @@ int main()
   auto tableClient = TableClient::CreateFromConnectionString(GetConnectionString(), TableName);
 
   // create new table
-  tableClient.Create();
+  tableServiceClient.CreateTable(TableName);
```

Comment:
Does this function return a value? Is it a best practice to check the return value, or is it going to throw an exception on failure?

Reply:
throw